rskj-core/src/test/java/co/rsk/test/builders/P2shErpFederationBuilder.java

```diff
@@ -0,0 +1,79 @@
+package co.rsk.test.builders;
```

Comment:
Let's better put this in `federation` package. I know other builder are here but shouldn't really be the case

Reply:
ok!